pinnwand/http.py

```diff
@@ -86,7 +89,8 @@ def check_xsrf_cookie(self) -> None:
 
 
 class ShowPaste(Base):
-    async def get(self, paste_id: str) -> None:
+    async def get(self, path: str) -> None:
+        paste_id, _, description = path.partition("#")  # ignore anchor part
```

Comment:
What would be correct way to process url here? Scanned tornado.routing docs, didn't found any.

Reply:
Uh, did some regex magic in urlspec.